toolchain/check/handle_class.cpp

```diff
@@ -512,6 +512,15 @@ auto HandleParseNode(Context& context, Parse::BaseDeclId node_id) -> bool {
     return true;
   }
 
+  if (!context.struct_type_fields_stack().PeekArray().empty()) {
```

Comment:
```suggestion
  if (auto fields = context.struct_type_fields_stack().PeekArray(); !fields.empty()) {
```

(see note suggestion below)